tests/infra/base_smoketest.py

```diff
@@ -198,7 +198,12 @@ def _test_replica_sync(self, current_replica, bundle_uuid, ):
                 else:
                     parser.exit(RED(f"Failed to replicate bundle from {current_replica.name} to {replica.name}"))
             with self.subTest(f"{current_replica.name}: Download bundle from {replica}"):
-                run(f"{self.venv_bin}hca dss download --replica {replica.name} --bundle-uuid {bundle_uuid}")
+                for i in range(30):
+                    try:
+                        run(f"{self.venv_bin}hca dss download --replica {replica.name} --bundle-uuid {bundle_uuid}")
+                        break
+                    except:
+                        time.sleep(1)
```

Comment:
can we throw a print statement here indicating there was an error?

Reply:
The CLI has built in retries. If there is a problem, this may take much longer than 30 seconds.